core/src/main/scala/com/nvidia/spark/rapids/tool/qualification/PluginTypeChecker.scala

```diff
@@ -325,44 +354,25 @@ class PluginTypeChecker(platform: Platform = PlatformFactory.createInstance(),
   }
 
   def getSpeedupFactor(execOrExpr: String): Double = {
-    supportedOperatorsScore.get(execOrExpr).getOrElse(1)
+    supportedOperatorsScore.get(execOrExpr).getOrElse(1.0)
   }
 
   def isExecSupported(exec: String): Boolean = {
     // special case ColumnarToRow and assume it will be removed or will we replace
     // with GPUColumnarToRow. TODO - we can add more logic here to look at operator
     //  before and after
+    // TODO: This is very tricky that we have this special case handled here.
```

Comment:
@nartal1 Do you know why do we check the "ColumnarToRow" here? This is the first time I notice that in this class.
Obviously, this check is never called because the tool marks `ColumnarToRow` as unsupported.
Any ideas?

Reply:
I am not certain why we have this check. It could be that we had this check before i.e. when we didn't have `shouldRemove` flag. So the code would always return true if there is a ColumnarToRow transition because this transition should not happen in the plugin. I don't see any merit in having this check here.

Reply:
I removed it and fixed the unit tests.